ClinGen/clincoded
# Connects to #1184. Provisional classification summary table

Adds this second table to the provisional curation page:

![image](https://cloud.githubusercontent.com/assets/4326866/21835780/8301c428-d774-11e6-8d71-837145b9c0e2.png)

Testing:

1. Create GDM
2. While there is no Evidence, press the Generate Summary button to and confirm that the scores are 0 and 'Current Saved Provisional Classification' is 'None'.
3. Add evidence to GDM to bring total score to 6.
4. Get to Summary page and confirm that the 'Limited' section is highlighted in blue. Confirm that clicking the 'Replication Over Time' checkbox does not affect the classification.
5. Save the classification and navigate back to the Summary to confirm that the 'Current Saved Provisional Classification' is correct.
6. Add evidence to GDM to bring total score to 6.5, and repeat steps 4-5, except confirm that 'Moderate' section is highlighted in blue (round up)
7. Add evidence to GDM to bring total score to 11.5, and repeat steps 4-5, except confirm that 'Strong' section is highlighted in blue (round up)
8. While calculated classification is at 'Strong', try pressing the 'Replication Over Time' checkbox and confirm that this bumps up the automatic classification to 'Definitive'. Confirm that this saves properly.
9. Modify the classification w/ no Explanation. Confirm that you cannot save and an error is generated.
10. Modify the classification w/ Explanation. Confirm that this saves and that the 'Current Saved Provisional Classification' is correct.
